src/main/java/cloud/fogbow/ras/core/plugins/interoperability/azure/quota/AzureQuotaPlugin.java

```diff
@@ -0,0 +1,296 @@
+package cloud.fogbow.ras.core.plugins.interoperability.azure.quota;
+
+import cloud.fogbow.common.constants.AzureConstants;
+import cloud.fogbow.common.constants.FogbowConstants;
+import cloud.fogbow.common.exceptions.FogbowException;
+import cloud.fogbow.common.models.AzureUser;
+import cloud.fogbow.common.util.AzureClientCacheManager;
+import cloud.fogbow.common.util.PropertiesUtil;
+import cloud.fogbow.ras.api.http.response.quotas.ResourceQuota;
+import cloud.fogbow.ras.api.http.response.quotas.allocation.*;
+import cloud.fogbow.ras.core.plugins.interoperability.QuotaPlugin;
+import com.google.common.annotations.VisibleForTesting;
+import com.microsoft.azure.PagedList;
+import com.microsoft.azure.management.Azure;
+import com.microsoft.azure.management.compute.ComputeUsage;
+import com.microsoft.azure.management.compute.Disk;
+import com.microsoft.azure.management.compute.VirtualMachine;
+import com.microsoft.azure.management.compute.VirtualMachineSize;
+import com.microsoft.azure.management.network.NetworkUsage;
+
+import javax.validation.constraints.NotNull;
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class AzureQuotaPlugin implements QuotaPlugin<AzureUser> {
+
+    private static final String QUOTA_VM_INSTANCES_KEY = "virtualMachines";
```

Comment:
Can these constants be used by other plugins? If yes, you could move it to the Common Project

Reply:
They are used only in the quota context.